Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.6
```

Comment:
done, I had to install git and change the tar command, but the resulting image should still be smaller.